src/ui/components/DevTools.js

```diff
@@ -109,7 +110,7 @@ function DevTools({
     return null;
   }
 
-  if (queryIsLoading) {
+  if (isLoading() || queryIsLoading) {
```

Comment:
If the whole URL is `/view?loading=true`, I don't think we'll be rendering the `DevTools` component at all, so I think this needs to be handled in `App.js`?